db/migrate/20160111155704_add_rating_to_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class AddRatingToReviews < ActiveRecord::Migration
+  def change
+    add_column :reviews, :rating, :integer
```

Comment:
Remove this attribute, you're going to have to implement a rating/voting system that will require additional tables
 Add a line note